topics/single-cell/tutorials/scrna-preprocessing-tenx/workflows/scRNA-seq-Preprocessing-TenX.ga

```diff
@@ -0,0 +1,457 @@
+{
+    "a_galaxy_workflow": "true",
+    "annotation": "",
+    "creator": [
+        {
+            "class": "Organization",
```

Comment:
```suggestion
            "class": "Person",
```
you're a person, not an organisation.

Additionally please credit @nomadscientist since she wrote the original workflow that you're updating here.

Reply:
sure, added all the authors from the training material